.github/workflows/run_pr_benchmarks.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This script is responsible for cloning a specific branch (the PR branch) from a GitHub repository and running Java Microbenchmark Harness (JMH) benchmarks.
+
+# Change directory to the one named after the branch name. 
```

Comment:
All the comments from this one on in this script are unnecessary and can be deleted